README.md

```diff
@@ -3,6 +3,7 @@
 [RethinkDB](https://www.rethinkdb.com)
 ======================================
 
+[![Coverage Status](https://coveralls.io/repos/github/RebirthDB/rebirthdb/badge.svg?branch=next)](https://coveralls.io/github/RebirthDB/rebirthdb?branch=next)
```

Comment:
@grandquista, if it won't be too far outside this PR's scope, would you add the Travis build status badge too?